### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by mbdesigns1989**

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+name: Linters
```
Comment (rloterh) [inline .github/workflows/linters.yml:1]: - [ ] Your Stylint linter is not properly configured. Tests are failing. Kndly follow [this link](https://github.com/microverseinc/linters-config/tree/master/html-css) to ensure that stylint is properly installed and that it's not failing during checks.

File: README.md [hunk 2]
```diff
@@ -1 +1,73 @@
...
+## Built With
+
+- Html
+- Css
+- Vscode
+- linters
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
```
Comment (rloterh) [inline README.md]: - [ ] This demo link is not working correctly. Please use [Github Pages](https://pages.github.com/) to host your demo project and provide the link accordingly at this part of the ReadMe.

File: README.md [hunk 3]
```diff
@@ -1 +1,73 @@
...
\ No newline at end of file
+# Hello-Microvese
+
+- First Html project in Microverse.
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Screen Shot
+
+![screenshot](./app_screenshot.png)
```
Comment (rloterh) [inline README.md]: - [ ] The screenshot link in your ReadMe is broken. Kindly fix it.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi MB Taha, 

Welcome to your first task😄, you're doing good!👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there!👍 

## Required Changes ♻️
- [ ] _Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
__As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).__

**Review by hemantso — APPROVED** [review]
Hi @mbdesigns1989, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 

Alt Text

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Please delete the Prerequisites in your `README.md` file that you are not using.
- Please make sure to use the uppercase letter for `commit messages`

Cheers,
Hemant Soni
Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**